edi_backend_partner_oca/__manifest__.py

```diff
@@ -8,7 +8,7 @@
     "license": "LGPL-3",
     "author": "ForgeFlow, Odoo Community Association (OCA)",
     "maintainers": ["LoisRForgeFlow"],
-    "development_status": "Beta",
+    "development_status": "Alpha",
```

Comment:
yeah, we could probably switch the base module to alpha (pls do it in its own commit)